lib/data/external/GCP/GcpUtils.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
placeholder?

Reply:
Yes. GcpUtils at the moment only contains a placeholder. I plan on using this file for shared objects/functions.

Reply:
I had that ideas as well, but I went with the separate `GcpUtils` file to set apart the code for GCP API implementation and client backend utilities